tests/lib/rules/indent.js

```diff
@@ -829,7 +842,7 @@ ruleTester.run("indent", rule, {
                 [189, 2, 0, "VariableDeclaration"],
                 [193, 6, 4, "ExpressionStatement"],
                 [195, 6, 8, "ExpressionStatement"],
-                [197, 2, 0, "VariableDeclaration"],
+                [196, 2, 4, "VariableDeclaration"],
```

Comment:
Oh, this is part of the bug fix. It was reporting the wrong values for the fixture. Go check yourself, this line and indentation levels for were for the parent if block's closing brace :)